### DISCUSSION TITLE
Does LYTV Crash when selecting a STAR with the FBW A320?

### DISCUSSION THREAD

**Discussion by 0x070E** [flybywiresim/aircraft#6061] [Help]
Hey there just a question before im going to file a issue.

A friend and i tried to fly to Tivat LYTV but when selecting a STAR the whole Sim freezes ... only seems to struggle with the FBW A320, maybe you can X Check that.  THX

**Comment by jcc1953** [comment]
Hello,
I have the same problem with MHLM airport but if I remove the mod  (SierraSim) for this airport there is no more problem. So I think you have a mod for LYTV and perhaps there is non compability with this mod and the last Airac.
